### PR TITLE
Add @githubteacher bio to bios folder

### PR DESCRIPTION
**Pull request by githubteacher**
I added a bio for @githubteacher as requested in #6.

Please take a :eyes: at this @jaw6.  Thank you for your input! :smiley:

### CODE REVIEW

**Comment by jaw6** [comment]
:+1: This looks good to me!

**Comment by githubteacher** [comment]
Everything looks good to me! :+1:

File: bios/githubteacherbio.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# GitHub Teacher
+
+### Interests
+- teaching
```
Comment (githubteacher) [inline bios/githubteacherbio.md]: I think this line should be capitalized for better formatting consistency.

**Comment by ateichmann** [comment]
:+1:

**Comment by githubteacher** [comment]
:+1:

**Pull request merged**